src/msw/webview_ie.cpp

```diff
@@ -104,21 +120,21 @@ bool wxWebViewIE::Create(wxWindow* parent,
     m_webBrowser->put_RegisterAsBrowser(VARIANT_TRUE);
     m_webBrowser->put_RegisterAsDropTarget(VARIANT_TRUE);
 
-    m_uiHandler = new DocHostUIHandler(this);
+    m_uiHandler = new DocHostUIHandler(m_webview);
 
-    m_container = new wxIEContainer(this, IID_IWebBrowser2, m_webBrowser, m_uiHandler);
+    m_container = new wxIEContainer(m_webview, IID_IWebBrowser2, m_webBrowser, m_uiHandler);
 
     EnableControlFeature(21 /* FEATURE_DISABLE_NAVIGATION_SOUNDS */);
 
-    // Make behaviour consistent with the other backends when loading localhost
-    // pages without any physical network connection.
-    SetOfflineMode(false);
-
-    LoadURL(url);
     return true;
 }
 
 wxWebViewIE::~wxWebViewIE()
+{
+    wxDELETE(m_impl);
```

Comment:
Really a nitpick, but why not
```suggestion
    delete m_impl;
```
We don't care about its future value in the dtor...